example/package.json

```diff
@@ -3,7 +3,8 @@
   "version": "1.0.0",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "extract:app": "unzip -o ./wikipedia_ios.zip -d wikipedia_ios.app && mv  wikipedia_ios.app/Wikipedia.app ./wikipedia.app && pwd && rm -rf wikipedia_ios.app"
```

Comment:
should we move this to a nodejs script ? 

create a folder called `tools` and keep a nodejs script called `extract.js` 

and use that to do this job ? we can atleast have support for logging there.